packages/manager/apps/dedicated/client/app/components/dedicated-cloud/service-pack/upgrade/steps/place-order/confirm/confirm.controller.js

```diff
@@ -36,4 +40,21 @@ export default class {
     this.monthly =
       this.prices.monthly.exists && this.prices.monthly.display.substr(1);
   }
+
+  onValide() {
+    this.atInternet.trackClick({
+      name: `dedicated::dedicatedCloud::details::servicePackUpgrade::basicOptions::confirm_${this.itemRef}`,
```

Comment:
Can we not use the already available `trackClick` function  declared in `dedicatedCloud-datacenters.routes.js`?

Reply:
I think the scope is not the same because I can't get the trackClick function.
Place-order/confirm/ haven't components for bindings.
I went for simplicity.